src/details/ArborX_Callbacks.hpp

```diff
@@ -20,6 +20,13 @@
 
 namespace ArborX
 {
+
+enum class CallbackTreeTraversalControl
```

Comment:
Any opinion on the name of the enum?  It might be less relevant with the `BruteForce` data structure or a binning strategy.

Reply:
I would change it to `CallbackControl` or similar, making it more general than just a tree.

Reply:
I don't like it because it does not indicate what it controls and could even mislead into thinking intros the callback.

Reply:
Not going to bikeshed, it's acceptable.